config/routes.rb

```diff
@@ -1,26 +1,31 @@
 Myflix::Application.routes.draw do
   root to: "pages#front"
-  get 'home', to: "videos#index"
-  get 'register', to: "users#new"
-  get 'sign_in', to: "sessions#new"
-  get 'sign_out', to: "sessions#destroy"
-  get 'my_queue', to: "queue_items#index"
-  post 'update_queue', to: "queue_items#update_queue"
+  get "home", to: "videos#index"
+  get "register", to: "users#new"
+  get "sign_in", to: "sessions#new"
+  get "sign_out", to: "sessions#destroy"
+  get "my_queue", to: "queue_items#index"
+  post "update_queue", to: "queue_items#update_queue"
 
   resources :queue_items, only: [:create, :destroy]
-  get 'ui(/:action)', controller: 'ui'
+  get "ui(/:action)", controller: 'ui'
 
-
-  resources :videos, only: [:show, :index] do 
-    collection do 
+  resources :videos, only: [:show, :index] do
+    collection do
       get :search, to: "videos#search"
     end
     resources :reviews, except: [:edit, :update, :destory]
   end
 
-  get 'people', to: 'relationships#index'
+  get "people", to: 'relationships#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.